pages/index.js

```diff
@@ -11,7 +13,17 @@ export default function Home() {
       </Head>
 
       <main>
-        <h1>My App</h1>
+        <h1>Your Daily Flow</h1>
+
+        {asanas.map((asana) => {
+          return (
+            <AsanaCard
+              key={asana.id}
+              name={asana.english_name}
+              img={asana.img_url}
+            />
+          );
+        })}
```

Comment:
Your Asanas should be inside an ul and be li elements, sorry if my last comment was confusing.

Reply:
Ah ok. No problem. I changed it.